run_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Run unit tests
```

Comment:
No need for the comment, there's a big nice string in the next line that tells you already what this is.